exp/util/util.go

```diff
@@ -79,7 +79,7 @@ func MachinePoolToInfrastructureMapFunc(gvk schema.GroupVersionKind, log logr.Lo
 			return nil
 		}
 
-		log.V(4).Info("projecting object", "namespace", m.Namespace, "name", ref.Name)
+		log.V(4).Info("projecting object", "Namespace", m.Namespace, "Name", ref.Name)
```

Comment:
Question that needs an answer to ensure consistency:
Should namespace and name be capitalised? (or either or neither 🙂)

Reply:
Made another suggestion. I think we can get rid of this entirely